auth-web/src/components/auth/mixins/TeamManagementMixin.vue

```diff
@@ -68,14 +69,16 @@ export default class TeamManagementMixin extends Vue {
     this.modal = confirmActionDialog
     if (member.membershipStatus === MembershipStatus.Pending) {
       this.confirmActionTitle = this.$t('confirmDenyMemberTitle').toString()
-      this.confirmActionText = `Are you sure you want to deny membership to ${member.user.firstname}?`
+      this.confirmActionText = `Deny account access to ${member.user.firstname} ${member.user.lastname}?`
       this.confirmHandler = this.deny
       this.primaryActionText = 'Deny'
+      this.primaryActionType = 'error'
     } else {
       this.confirmActionTitle = this.$t('confirmRemoveMemberTitle').toString()
-      this.confirmActionText = `Are you sure you want to remove team member ${member.user.firstname} ${member.user.lastname} from this account?`
+      this.confirmActionText = `Are you sure you want to remove team member <strong>${member.user.firstname} ${member.user.lastname}</strong> from this account?`
```

Comment:
same as above

Reply:
As above -  that's how it was done originally. Can we do this as a separate task if in fact it's needed?